controlplane/pkg/plugins/utils.go

```diff
@@ -38,6 +39,32 @@ func StartPlugin(name, registry string, services map[plugins.PluginCapability]in
 		return err
 	}
 
+	go func() {
+		registered := true
+		for ; ; time.Sleep(5 * time.Second) {
+			resp, err := http.Get("http://localhost:5555/liveness")
```

Comment:
Let's create something like `probesClient` in package `probes` to avoid dependencies on specific port and URL path

Reply:
CheckNsmdLiveness() function is defined in probes package to encapsulate the code for checking NSMD liveness.